### ISSUE TITLE
potionWhiteList resets to its original value

### ISSUE THREAD

**Issue by maksim10411072** [MightyPirates/OpenComputers#3555]
I found it in the log:
```
[16:54:08] [Client thread/INFO]: Updating config from version '@VERSION@' to '@VERSION@'.
[16:54:09] [Client thread/INFO]: Updating setting 'opencomputers.switch.relayDelayUpgrade'. 
[16:54:09] [Client thread/INFO]: Updating setting 'opencomputers.nanomachines.potionWhitelist'. 
[16:54:09] [Client thread/INFO]: Updating setting 'opencomputers.power.cost.wirelessCostPerRange'. 
[16:54:09] [Client thread/INFO]: Updating setting 'opencomputers.misc.maxWirelessRange'. 
[16:54:09] [Client thread/INFO]: Updating setting 'opencomputers.misc.maxOpenPorts'. 
[16:54:09] [Client thread/INFO]: Updating setting 'opencomputers.computer.cpuComponentCount'. 
```

**Comment by asiekierka** [comment]
Isn't this a one-off issue in OpenComputers 1.7.6, which should be fixed by OpenComputers 1.7.7?

**Comment by maksim10411072** [comment]
> 

upgrading to 1.7.7 helped, thanks
